recipes/lsdtt-xtensor-python/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "lsdtt-xtensor-python" %}
+{% set version = "0.0.4.10" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lsdtt_xtensor_python-{{ version }}.tar.gz
+  sha256: c09a4f2c91ad0e4de6a71c1704f5e8cad003258dd40d73c3206b4522dee1ebe6
+
+build:
+  number: 0
+  skip: true   # [py<37]
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - xtensor 
+    - xtensor-python 
+    - numpy >=1.14  # [linux]
+    - numpy  # [not linux]
```

Comment:
please see https://conda-forge.org/docs/maintainer/knowledge_base.html#building-against-numpy

Reply:
Thanks @scopatz !
I think it should be sorted. I got odd, inconsistent errors for building for OSx: the conda-forge-ci-setup package was not found